isis/src/base/objs/SerialNumber/SerialNumber.cpp

```diff
@@ -122,10 +122,7 @@ namespace Isis {
     if(translationIterator == missionTranslators.end()) {
       // Get the file
 
-      FileName snFile((QString) "$ISISROOT/translations/" + mission + instrument + "SerialNumber????.trn");
-      std::cout << "SERIALNUMBER: " << snFile.baseName() << std::endl; 
-
-      snFile = snFile.highestVersion();
+      FileName snFile((QString) "$ISISROOT/appdata/translations/" + mission + instrument + "SerialNumber.trn");
```

Comment:
Yikes, bet that was causing a few failures

Reply:
Not sure how many of the 150 tests that aren't failing now are due to this, but Yikes indeed!